### PR TITLE
Update aoba.pug

### PR DESCRIPTION
**Pull request by TheSmallBlue**

### CODE REVIEW

File: src/pug/characters/aoba.pug [hunk 1]
```diff
@@ -6,16 +6,19 @@ include ../templates
...
   },
   strengths: [
     "Is adorable",
-    "Is a legal loli",
-    "Can do the art thing",
-    "Has occasional sadism"
+    "Doesn't drink nor smoke",
+    "Very good at art",
+    "Has a bear sleeping bag",
+    "Would and SHOULD have lesbo sex with Hifumi"
```
Comment (Xetera) [inline src/pug/characters/aoba.pug]: no

File: src/pug/characters/aoba.pug [hunk 2]
```diff
@@ -6,16 +6,19 @@ include ../templates
...
   },
   strengths: [
     "Is adorable",
-    "Is a legal loli",
-    "Can do the art thing",
-    "Has occasional sadism"
+    "Doesn't drink nor smoke",
+    "Very good at art",
+    "Has a bear sleeping bag",
+    "Should absolutley go out with hifumi they're such a cute couple"
```
Comment (itsaferbie) [inline src/pug/characters/aoba.pug]: Can you get the capitalization right at least?

Comment (jvyden420) [reply src/pug/characters/aoba.pug]: ^

Comment (TheSmallBlue) [reply src/pug/characters/aoba.pug]: Fixerino

File: src/pug/characters/aoba.pug [hunk 3]
```diff
@@ -6,16 +6,19 @@ include ../templates
...
   },
   strengths: [
     "Is adorable",
-    "Is a legal loli",
-    "Can do the art thing",
-    "Has occasional sadism"
+    "Doesn't drink nor smoke",
+    "Very good at art",
+    "Has a bear sleeping bag",
+    "Should absolutley go out with Hifumi, they're such a cute couple"
```
Comment (Xetera) [inline src/pug/characters/aoba.pug]: still wrong, also this isn't a strength

Comment (jvyden420) [reply src/pug/characters/aoba.pug]: w-which one? the not drinking or smoking or the not going out with Hifumi?

Comment (Xetera) [reply src/pug/characters/aoba.pug]: absolutley

Comment (whamer100) [reply src/pug/characters/aoba.pug]:
```suggestion
-    "Should absolutley go out with Hifumi, they're such a cute couple"
```

Comment (whamer100) [reply src/pug/characters/aoba.pug]:
```suggestion
    
```

File: src/pug/characters/aoba.pug [hunk 4]
```diff
@@ -6,16 +6,19 @@ include ../templates
...
   },
   strengths: [
     "Is adorable",
-    "Is a legal loli",
-    "Can do the art thing",
-    "Has occasional sadism"
+    "Doesn't drink nor smoke",
+    "Very good at art",
+    "Has a bear sleeping bag",
+    "Has the possibilty of going out with Hifumi and she should absolutley take it, they're such a cute couple"
```
Comment (itsaferbie) [inline src/pug/characters/aoba.pug]: You know what needs to be done.

**Comment by TheSmallBlue** [comment]
Aight, im done

**Comment by TheSmallBlue** [comment]
hewwo swenpai cowwld you pwease acccwept my puww reqwest uwu

**Pull request closed without merging**